examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -165,12 +165,12 @@
     "from pyspark.sql.types import StructType, StructField\n",
     "from pyspark.sql.types import StringType, FloatType, IntegerType, LongType\n",
     "\n",
-    "from recommenders.utils.timer import Timer\n",
-    "from recommenders.datasets import movielens\n",
-    "from recommenders.utils.notebook_utils import is_jupyter\n",
-    "from recommenders.datasets.spark_splitters import spark_random_split\n",
-    "from recommenders.evaluation.spark_evaluation import SparkRatingEvaluation, SparkRankingEvaluation, SparkDiversityEvaluation\n",
-    "from recommenders.utils.spark_utils import start_or_get_spark\n",
```

Comment:
hey Yan, can you please revert this change. We renamed reco_utils to recommenders, see this PR: https://github.com/microsoft/recommenders/pull/1485 this is why the test it is failing. 

Also, you need to reinstall recommenders in your local machine: `pip install -e .[all]`